lib/screens/product_page/product_page.dart

```diff
@@ -21,12 +33,16 @@ class ProductPage extends StatelessWidget {
   Widget build(BuildContext context) {
     // POUR LES PHASES DE DEV , NE PAS SUPPRIMER
     // final ProductModel product =
-    //     ProductModel.fromMock().firstWhere((e) => "2" == e.id);
+    //     ProductModel.fromMock().firstWhere((e) => "1" == e.id);
     final ProductModel product = ModalRoute.of(context).settings.arguments;
```

Comment:
Why do you use final?

Reply:
final because the variable is init once and for all. never changed. for optimisation purpose and to understand this variable is never change.